src/usb_cam_node.cpp

```diff
@@ -198,6 +206,20 @@ bool UsbCamNode::take_and_send_image()
   return true;
 }
 
+rcl_interfaces::msg::SetParametersResult UsbCamNode::parametersCallback(
+  const std::vector<rclcpp::Parameter> & parameters)
+{
+  RCLCPP_INFO(get_logger(), "Callback called!");
```

Comment:
@progtologist if we do want to leave this print in here, maybe make it more informative and make it a debug statement?

```suggestion
  RCLCPP_DEBUG(get_logger(), "Setting parameters for %s", camera_name_.c_str());
```

Adding the camera_name since the user might have multiple cameras running

Reply:
Oops, yeah, forgot this during debugging :$
We can remove it or do it as you suggest :)